Doc/library/unittest.mock.rst

```diff
@@ -1299,13 +1299,21 @@ patch.dict
     decorator. When used as a class decorator :func:`patch.dict` honours
     ``patch.TEST_PREFIX`` for choosing which methods to wrap.
 
+    .. versionchanged:: 3.7
```

Comment:
This could be updated since it's going in for master now.

```suggestion
    .. versionchanged:: 3.8
```

Reply:
This still applies!